File: LeetCode/Tag/Two Pointers/java/80.remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i = 0;
        for (int j = 0; j < nums.length; ++j) {
            if (i < 2 || nums[j] > nums[i-2])
                nums[i++] = nums[j];
        }
        return i;
    }
}

File: LeetCode/Difficulty/Medium/js/80.remove-duplicates-from-sorted-array-II.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function (nums) {
    let i = 0, j = 1, count = 1, arrLen = nums.length;
    for (; j < arrLen; j++) {
        if (nums[i] === nums[j]) {
            if (count < 2) {
                i++;
                nums[i] = nums[j];
            }
            count++;
        } else {
            i++;
            nums[i] = nums[j];
            count = 1;
        }
    }
    return i + 1;
};